inverse_covariance/tests/rank_correlation_test.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+from sklearn.utils.testing import assert_array_almost_equal
+
+from inverse_covariance.rank_correlation import (
+    _compute_ranks,
+    spearman_correlation,
+)
+
+Y = np.ones(shape=[10, 2])
+X = .001 * np.random.randn(20, 1)
+X = np.append(X, X, axis=1)
+
+
+def test_compute_ranks():
+    Y1 = 5.5 * Y
+    Y2 = _compute_ranks(Y)
+    assert_array_almost_equal(Y1, Y2)
+
+
+def test_spearman_correlation():
```

Comment:
I know that many of the tests I've provided as examples for the other files just test the integrity of the interface (inputs/outputs/shapes) but if you also have some examples of testing for "correctness" in any of the new stuff you port over, that would be great too.